src/main/java/DukeCommands.java

```diff
@@ -0,0 +1,54 @@
+public class DukeCommands {
+    public static String spacer = "__________________________________________\n";
+    public static DukeTasks[] list = new DukeTasks[100];
```

Comment:
A possible slight improvement that you can consider is using ArrayList so there is no need to specific a fixed size